tool/grind.dart

```diff
@@ -899,12 +899,13 @@ Future<void> testDartdocRemote() async {
       [object]);
 }
 
-@Task('serve docs for a package that requires flutter with remote linking')
-@Depends(buildDartdocFlutterPluginDocs)
-Future<void> serveDartdocFlutterPluginDocs() async {
-  await _serveDocsFrom(
-      pluginPackageDocsDir.path, 8005, 'serve-dartdoc-flutter-plugin-docs');
-}
+// TODO(gspencer): Re-enable this as soon as Flutter's config is sane again.
```

Comment:
You should be able to do this more cleanly by removing the `testDartdocFlutterPlugin `reference from 'validateFlutterDocs', which is what is run by the travis.  I don't think this will actually have any impact on the travis, it will still try to run a plugin build.

Reply:
Oh, that's much better.  Done.